Extensions/ParticleSystem/Extension.cpp

```diff
@@ -15,37 +15,162 @@ This project is released under the MIT License.
 #include "Extension.h"
 
 
-/**
- * Constructor of an extension declares everything the extension contains: objects, actions, conditions and expressions.
- */
-Extension::Extension()
+void DeclareParticleSystemExtension(gd::PlatformExtension & extension)
```

Comment:
This is indeed what you needed to do, as done in other extensions that are available for both GDCpp and GDJS 👍